### DISCUSSION TITLE
Adding interests to the account

### DISCUSSION THREAD

**Discussion by ghost** [ghostfolio/ghostfolio#874] [Ideas]
Hi! I'm having a bit of trouble trying to understand how to do this correctly. Let's say I have 1000 UST in TerraStation with the current APY. I log every week how much I got from staking it in Anchor, which would be around 3.5UST. 

I wouldn't like to add it as a Buy, since I'm not actually buying anything, it's just the staking process that's rewarding me with more tokens, somewhat similar to a Dividend. I tried using it, but it won't add the new amount to the amount in the account and it won't impact in my Dashboard's performance. 

I think ideally I should be able to add an APY to a staking account or at least being able to add the staking reward manually and see it as a positive % in my dashboard and my account.

**Comment by dtslvr** [comment]
Thanks for sharing your ideas @francocalvo.

For P2P credits with fixed income, I track the interest as dividend. But it is in EUR and it is not accumulating.

The total dividend (as an absolute number) is currently only available in the summary. I think you are addressing multiple topics here:

- Calculate the dividend yield
    - Total dividend yield for a single position
    - Include the dividend yields of all positions in the overall performance
- Treat cryptocurrencies as fiat currencies (e.g. set as currency in account)
- …

Do you agree with such an approach? Are you interested in implementing one of these topics?

**Comment by ghost** [comment]
Hi! I'd say that's pretty accurate. 

- Cryptocurrencies should be treated as fiat, as they can be exchanged to EUR or any other fiat currency besides dollars directly.
- Yields should be seen in the overall performance. It's not in my use-case, but there are companies that are very dividends oriented.
- With crypto, staking is becoming somewhat important. It's making your position bigger in the cryptocurrency, regardless of its price. So if I had 100UST and got 19UST in a year, I'd like to see my account with 119UST, which would be the accurate number.

I'd like to help! I'm a junior dev with knowledge in Python and a bit of Javascript. I saw that you are working in Typescript. I'm not yet in a position where I could design and implement it alone even if I knew the codebase, which I don't, but I can take on smaller tasks and work with someone more experienced.

**Reply by dtslvr** [reply]
Have you already been able to set up the project locally? Please join the [Slack Community](https://join.slack.com/t/ghostfolio/shared_invite/zt-vsaan64h-F_I0fEo5M0P88lP9ibCxFg) if you need support and to discuss the next steps.
